### ISSUE TITLE
Object of type 'Camera' has been destroyed

### ISSUE THREAD

**Issue by dershver** [microsoft/MixedRealityToolkit-Unity#9200]
My Unity project has two scenes. In the second scene, I get a continuous error that the Object of type Camera has been destroyed.
The exact text of the error is:
MissingReferenceException: The object of type 'Camera' has been destroyed but you are still trying to access it.
Your script should either check if it is null or you should not destroy the object.

This happens whenever a configure my second scene to use MRTK. If MRTK is not present, there are no problems.
The first scene uses MRTK and function properly.

## Expected behavior

The camera object should not be destroyed. There is no code in my script for this scene that deletes the camera.

## Screenshots

This is a screenshot of the GameObjects in the scene after the error occurs. The Camera object name 
'Acuity Camera' is still present.

![AcuitySceneMRTK](https://user-images.githubusercontent.com/54962413/106518461-eccb6200-64a7-11eb-988e-99c06d4ede94.JPG)

- Unity Version 2019.4.16f1
- MRTK Version 2.3.0

## Target platform (please complete the following information)

- HoloLens 2

## Additional context

My code attempts to find this GameObject (Acuity Camera) and it fails.
 
 GameObject MyCam = GameObject.Find("Acuity Camera");
        if(MyCam == null)
        {
                  Debug.Log("Camera not found !!!!");
                   return;
        }

**Comment by Rive4** [comment]
Hello,
I am having the same problem. I am also trying to switch between scenes. Each one has its MixedRealityToolkit obj and the camera inside MixedReality Playspace. No scripts that delete the camera.

Did you find any solution for this?

**Comment by dershver** [comment]
I first attempted to use the Scene Manager in MRTK. This was not
simple. It took me awhile until I discovered where to
insert my 'content' scenes so they would switch properly. I had trouble
handling the Lighting scenes and it never worked well
in Play Mode. It always crashed on the HoloLens 2 due to an exception
related to the number of 'Hover Lights.'

      In the end, I created my own manager scene that had the Main Camera
and disabled the camera in the other scenes.
The manager scene loads the first scene as "Additive.' The other scenes
just switch among themselves loading/unloading each time.
The manager scene is always active and never unloaded.
     This was simple and very easy to implement. Let me know if you want
sample code.

                                                                 Paul Klahr
